deb/Dockerfile.stable-slim

```
@@ -5,7 +5,7 @@ ENV LC_ALL=C.UTF-8
 ENV PS1="(deb) \w \$ "
 
 RUN apt-get update && \
-    apt-get -y install bash ruby ruby-dev rubygems build-essential curl git zip bzip2 jq
+    apt-get -y -q install bash ruby ruby-dev rubygems build-essential curl git zip bzip2 jq
```

any reason not to use `-qq`? It also lets you drop the `-y`
```suggestion
    apt-get -qq install bash ruby ruby-dev rubygems build-essential curl git zip bzip2 jq
```

---

We still want logging in case something goes wrong, and `-qq` suppresses too much.